modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonFlaskConnexionServerCodegen.java

```diff
@@ -54,4 +54,11 @@ protected void addSupportingFiles() {
         supportingFiles.add(new SupportingFile("__init__.mustache", packagePath(), "__init__.py"));
         testPackage = packageName + "." + testPackage;
     }
+
+    @Override
+    public String toParamName(String name) {
+        // Connexion is broken and does not sanitize the parameter names.
+        // Since Connexion does no sanitization we cannot do it here sadly.
+        return name;
```

Comment:
What if the parameter name contains something like "-" (dash)? Would it still work without sanitization?